pkgs/applications/emulators/ryujinx/updater.sh

```diff
@@ -75,16 +75,4 @@ fi
 
 echo "building Nuget lockfile"
 
-STORE_SRC="$(nix-build . -A ryujinx.src --no-out-link)"
-SRC="$(mktemp -d /tmp/ryujinx-src.XXX)"
-cp -rT "$STORE_SRC" "$SRC"
-chmod -R +w "$SRC"
-pushd "$SRC"
-
-mkdir nuget_tmp.packages
-DOTNET_CLI_TELEMETRY_OPTOUT=1 dotnet restore Ryujinx.sln --packages nuget_tmp.packages
-
-nuget-to-nix ./nuget_tmp.packages >"$DEPS_FILE"
-
-popd
-rm -r "$SRC"
+$(nix-build -A ryujinx.fetch-deps --no-out-link) "$DEPS_FILE"
```

Comment:
Github wont let me comment on the line since its too high above any changed, but https://github.com/NixOS/nixpkgs/pull/187150/files#diff-83a0ea3f9e13f2d6a738a3b9e2bb99b828c78f03af844e9371b762232320cffdR2 should be changed to:
```shell
#! nix-shell -I nixpkgs=./. -i bash -p coreutils gnused curl common-updater-scripts nix-prefetch-git jq
```
The `fetch-deps` script already propagates the `nuget-to-nix` and the SDK.